smaug/python/smaug_test.py

```diff
@@ -82,4 +82,4 @@ def runAndValidate(self, graph, expected_output):
       sg_output = sg_output_proto.data.int64_data
     shape = _account_for_padding(sg_output_proto.shape)
     sg_output = np.reshape(sg_output, sg_output_proto.shape.dims)
-    assert_array_almost_equal(expected_output, sg_output, decimal=3)
+    assert_array_almost_equal(expected_output, sg_output, decimal=2)
```

Comment:
I would prefer to add decimal as a kwarg to runAndValidate, so that only the FP lossy tests need to lower it. We can use float32 for all the other tests to avoid precision loss if necessary.

Reply:
Done.